logpyle/runalyzer_gather.py

```diff
@@ -339,4 +341,8 @@ def transfer_data_table_multi(db_conn: Connection, tbl_name: str,
     pb.finished()  # type: ignore[no-untyped-call]
 
     db_conn.commit()
-    db_conn.close()
+    if return_conn:
+        return db_conn
+    else:
+        db_conn.close()
+        return None
```

Comment:
maybe always return the conn and expect the caller to close it?

Reply:
Currently the RunDB class closes the connection in its destructor